tests/aws/services/events/test_events_rules.py

```diff
@@ -1,345 +0,0 @@
-            "env": ["dev", "prod"],
-            "user": ["user1", "user2", "user3"],
-            "admins": ["skyli", {"prefix": "hey"}, {"prefix": "ad"}],
-            "test1": [{"anything-but": 200}],
-            "test2": [{"anything-but": "test2"}],
-            "test3": [{"anything-but": ["test3", "test33"]}],
-            "test4": [{"anything-but": {"prefix": "test4"}}],
-            # TODO: unsupported in LocalStack
-            # "ip": [{"cidr": "10.102.1.0/24"}],
-            "num-test1": [{"numeric": ["<", 200]}],
-            "num-test2": [{"numeric": ["<=", 200]}],
-            "num-test3": [{"numeric": [">", 200]}],
-            "num-test4": [{"numeric": [">=", 200]}],
-            "num-test5": [{"numeric": [">=", 200, "<=", 500]}],
-            "num-test6": [{"numeric": [">", 200, "<", 500]}],
-            "num-test7": [{"numeric": [">=", 200, "<", 500]}],
-        },
-    }
-
-    event = {
-        "EventBusName": event_bus_name,
-        "Source": "core.update-account-command",
-        "DetailType": "core.app.backend",
-        "Detail": json.dumps(
-            {
-                "description": "this-is-event-details",
-                "amount": 200,
-                "salary": 2000,
-                "env": "prod",
-                "user": "user3",
-                "admins": "admin",
-                "test1": 300,
-                "test2": "test22",
-                "test3": "test333",
-                "test4": "this test4",
-                "ip": "10.102.1.100",
-                "num-test1": 100,
-                "num-test2": 200,
-                "num-test3": 300,
-                "num-test4": 200,
-                "num-test5": 500,
-                "num-test6": 300,
-                "num-test7": 300,
-            }
-        ),
-    }
-
-    aws_client.events.create_event_bus(Name=event_bus_name)
-    aws_client.events.put_rule(
-        Name=rule_name,
-        EventBusName=event_bus_name,
-        EventPattern=json.dumps(pattern),
-    )
-
-    aws_client.events.put_targets(
-        Rule=rule_name,
-        EventBusName=event_bus_name,
-        Targets=[{"Id": target_id, "Arn": queue_arn, "InputPath": "$.detail"}],
-    )
-    aws_client.events.put_events(Entries=[event])
-
-    messages = sqs_collect_messages(aws_client, queue_url, expected_events_count=1, retries=3)
-    assert len(messages) == 1
-    assert json.loads(messages[0].get("Body")) == json.loads(event["Detail"])
-    event_details = json.loads(event["Detail"])
-    event_details["admins"] = "no"
-    event["Detail"] = json.dumps(event_details)
-
-    aws_client.events.put_events(Entries=[event])
-
-    messages = sqs_collect_messages(
-        aws_client, queue_url, expected_events_count=0, retries=1, wait_time=3
-    )
-    assert messages == []
-
-    # clean up
-    clean_up(
-        bus_name=event_bus_name,
-        rule_name=rule_name,
-        target_ids=target_id,
-        queue_url=queue_url,
-    )
-
-
-@markers.aws.validated
-@pytest.mark.skip
```

Comment:
I guess this test was flaky and therefore skipped 🧹 

It seems we are still missing tests for the CloudWatch target in `test_events_targets.py`, right?

Reply:
Yes the cloudwatch scenario is still not covered